docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,190 @@
+
+To begin the tutorial you will first install the Flocker client on your local machine, then install Flocker onto two hosts.
+You will then be ready to use Flocker to migrate a Docker container with a volume attached from one host to the other.
+
+You will be controlling your Flocker cluster via the CLI you've installed locally.
+The following diagram illustrates the initial server-side Flocker setup that you will control via the CLI:
+
+.. image:: images/flocker-tutorial-initial-setup.svg
+   :width: 60 %
+   :alt: In the initial server-side Flocker setup there are two servers, one of which has two Docker containers running; one container is a running a web application, the other has a Redis database with a volume.
+   :align: center
+
+The following diagram illustrates how the server-side Flocker setup will be configured at the end of the tutorial:
+
+.. image:: images/flocker-tutorial-final-setup.svg
+   :width: 60 %
+   :alt: Following the completion of this tutorial the server-side Flocker setup will be configured with the web application still running within a container on the first server, while the Redis server with a volume is running on the second server.
+   :align: center
+
+Flocker manages the data migration and the link between the two containers.
+
+If you have any feedback or problems, you can :ref:`talk-to-us`.
+
+Before You Begin
+================
+
+Requirements
+------------
+
+To replicate the steps demonstrated in this tutorial, you will need:
+
+* Ubuntu 14.04 or OS X with `Homebrew`_ installed.
+* Two Virtual Machines (VMs) on your local machine.
+  For this tutorial, you are supplied with Vagrant images to create the tutorial environment on VMs using VirtualBox, so you must have `Vagrant`_ and `VirtualBox`_ installed.
+* At least 4GB RAM available.
+
+Installing the Client
+=====================
+
+The Flocker client runs locally on your machine, and will control the two instances of Flocker located on the hosts.
+To install the Flocker client, run the following in your terminal:
+
+OS X
+----
+
+.. task:: test_homebrew flocker-|latest-installable|
+   :prompt: you@laptop:~$
+
+Ubuntu 14.04
+------------
+
+.. task:: install_cli ubuntu-14.04
+   :prompt: you@laptop:~$
+
+Check the Version
+-----------------
+
+To check the version of Flocker you have installed, run the following:
+
+.. prompt:: bash you@laptop:~$
+
+   flocker-deploy --version
+
+Installing Flocker on Local VMs
+===============================
+
+In this step, you will install two instances of Flocker, each on a separate host.
+Flocker manages the links, ports, and volumes associated with Docker containers and can move them around after deployment.
+
+#. Download the Vagrant configuration file, and the cluster and user credentials:
+
+   .. version-code-block:: console
+
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile
```

Comment:
It is weird that this is styled differently to the other prompts. It is weird that with some I can copy the whole lot and paste, with these I can't (the prompt gets copied). I understand that prompt can't be used here because there is no `version-prompt` so either:
* Use a different command which does the whole thing in one command (suggested by @exarkun on the original curl issue, @lukemarsden supported it)
* Link to the version-prompt issue https://clusterhq.atlassian.net/browse/FLOC-2141 with an XXX
* Wait for version-prompt and ask engineers to finish it (might not happen)